.about.yml

```diff
@@ -0,0 +1,105 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: project-dashboard
+
+# Full proper name of the project (required)
+full_name: '18F Project Dashboard'
```

Comment:
Wondering if we should elide `18F` from our project names. Since this app is part of our org, it's clearly an 18F dashboard. :-) Thoughts?